app.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express');
+const app = express();
+
+const methodOverride = require('method-override');
+const bp = require('body-parser');
+const exphbs = require('express-handlebars')
+
+const Products = require('./db/products.js');
+const DS_Prod = new Products();
+
+const Articles = require('./db/articles.js')
+const DS_Articles = new Articles();
+
+const PORT = process.env.PORT || 8000;
+
+app.use(express.static('public'));
+
+app.use(bp.urlencoded({ extended: true }));
+
+app.use(methodOverride('_method'));
+
+app.engine('.hbs', exphbs({ defaultLayout: 'main', extname: '.hbs' }));
+app.set('view engine', '.hbs');
+
+// GET - Home Page
+app.get('/', (req, res) => {
+    // res.send('IZ WORKING?')
+    res.render('home');
+});
+
+// ***** PRODUCTS ***** //
+
+//GET - Create New Products Form
+app.get('/products/new', (req, res) => {
+    res.render('form');
+})
+
+// GET - Products Page
+app.get('/products', (req, res) => {
+    const allProds = DS_Prod.all();
+    res.render('products', { allProds });
+});
+
+// GET Products by Id
+app.get('/products/:id', (req, res) => {
+    const { id } = req.params;
```

Comment:
`let`, is a signal that the variable may be reassigned, such as a counter in a loop, or a value swap in an algorithm. It also signals that the variable will be used only in the block it’s defined in, which is not always the entire containing function. - https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75